rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -209,6 +219,17 @@ void gs_frame::toggle_fullscreen()
 	});
 }
 
+void gs_frame::toggle_mouselock()
+{
+	// first we toggle the value
+	m_mouse_hide_and_lock = !m_mouse_hide_and_lock;
+
+	// and then we publish it to the window properties
+	this->setProperty("mouseLocked", m_mouse_hide_and_lock);
```

Comment:
Why are you suddenly using this-> in this function?

Reply:
For no good reason.  Just my mindset at the time of calling 'deeper' into the QWindow inheritance pattern.
For consistency I'll drop it back to just setProperty(...), removing the explicit this reference